canonical/juju-dashboard
# WD-6236 - Display controller status

## Done

- Display controller connection status.

## QA

- Go to the controller list.
- Use the register controller form to add a controller that doesn't exist (enter a fake address).
- In the table it should show a 'Disconnected' cell.
- Hover the cell and it should show the error.

## Details

https://warthogs.atlassian.net/browse/WD-6236
Fixes: https://github.com/canonical/juju-dashboard/issues/1625

## Screenshots

<img width="948" alt="Screenshot 2023-09-11 at 2 50 49 pm" src="https://github.com/canonical/juju-dashboard/assets/361637/3e752c02-bd7c-4e97-97f9-4df351f96dd6">
